padl/transforms.py

```diff
@@ -997,11 +1073,15 @@ def _pd_get_signature(self) -> List[str]:
             return inspect.signature(self).parameters
         return [f'arg_{i}' for i in range(self._pd_number_of_inputs)]
 
-    def _pd_longrepr(self, formatting=True) -> str:
+    def _pd_longrepr(self, formatting=True, marker=None) -> str:
         try:
-            return '\n'.join(self.source.split('\n')[:30])
+            str_ = self.source.split('\n')[:30]
+            if marker:
+                return str_[0] + marker[1] + '\n'.join(str_[1:])
+            return '\n'.join(str_)
         except TypeError:
-            return self._pd_call
+            return self._pd_call + marker[1] + '\n' if marker \
+                else self._pd_call
```

Comment:
I think you can put this all on one line.

Reply:
right